vajram/vajram-codegen/src/main/java/com/flipkart/krystal/vajram/codegen/VajramCodeGenerator.java

```diff
@@ -871,38 +871,42 @@ private void generateDependencyResolutions(
           } else {
             String message =
                 ("A resolver ('%s') must not access an optional dependencyDef ('%s') directly."
-                        + "Use Optional<>, Errable<>, or DependencyResponse<> instead")
+                        + "Use Optional<>, ValueOrError<>, or DependencyResponse<> instead")
```

Comment:
this is not my changes but upon merging with main and resolving conflicts its showing as my diff

Reply:
In main branch all usages of ValueOrError are replaced with Errable. Please do the same

Reply:
Done